lib/react.dart

```diff
@@ -214,31 +210,29 @@ abstract class Component {
 
   /// Allows the [ReactJS `displayName` property](https://reactjs.org/docs/react-component.html#displayname)
   /// to be set for debugging purposes.
-  String get displayName => runtimeType.toString();
+  String? get displayName => runtimeType.toString();
```

Comment:
TIL Dart's `toString` can return null?

Reply:
Oh it can't; this is actually nullable so that in the Component2 override can return null:
https://github.com/Workiva/react-dart/blob/dbcf43ad127a791f45277bea18fec2825fefa0fa/lib/react.dart#L677-L685

I can add a comment about this.

Reply:
Addressed in https://github.com/Workiva/react-dart/pull/373/commits/ee90271ade553281b32779002dbb03325e20db4d